docs/keyring-support.rst

```diff
@@ -0,0 +1,69 @@
+Keyring Support
+===============
+
+Instead of typing in your password every time you upload a distribution, Twine
+allows storing a username and password securely using `keyring`_.
+Keyring is installed with Twine but for some systems (Linux mainly) may
+require `additional installation steps`_.
+
+Once Twine is installed, use the ``keyring`` program to set a
+username and password to use for each package index (repository) to
+which you may upload.
+
+For example, to set a username and password for PyPI:
+
+.. code-block:: bash
+
+    keyring set https://upload.pypi.org/legacy/ your-username
+
+or
+
+.. code-block:: bash
+
+    python3 -m keyring set https://upload.pypi.org/legacy/ your-username
```

Comment:
Do we really need to demo how runpy works in each example? Maybe we could just have one command invocation block in each place (w/o `python -m` but put a note somewhere that it's also possible).

Reply:
Maybe add a note before example:

```rst

.. note:: 

    Adding ``python3 -m`` before each command here also works.
```

Reply:
I don't think the note is necessary; just the keyring command is fine.